dev-resources/puppetlabs/services/master/master_core_test/bolt_projects/local_afafaf/tasks/init.sh

```diff
@@ -0,0 +1,2 @@
+. $PT__installdir/helpers/files/marco.sh
```

Comment:
Does this task actually run without a shebang? (not that is needs to...)

Reply:
no idea, i just copied these out of bolt's tests